Sources/SWBCore/Specs/Tools/LinkerTools.swift

```diff
@@ -949,14 +949,14 @@ public final class LdLinkerSpec : GenericLinkerSpec, SpecIdentifierType {
         let argPrefix = "-Xlinker"
 
         // Args without parameters
-        for arg in ["-dynamiclib", "-bundle", "-r", "-fembed-bitcode-marker", "-dead_strip", "-nostdlib"] {
+        for arg in ["-dynamiclib", "-bundle", "-r", "-fembed-bitcode-marker", "-dead_strip", "-nostdlib", "-rdynamic"] {
             while let index = commandLine.firstIndex(of: arg) {
                 commandLine.remove(at: index)
             }
         }
 
         // Args without parameters (-Xlinker-prefixed, e.g. -Xlinker)
-        for arg in ["-bitcode_verify", "-export_dynamic"] {
```

Comment:
@neonichu Should this have been retained, in case anyone was passing `-export_dynamic` using `OTHER_LDFLAGS`?

Reply:
I guess that would make sense, yah

Reply:
https://github.com/swiftlang/swift-build/pull/107